modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js

```diff
@@ -1027,6 +1027,25 @@ var isExperimentTrackingSuccess = false;
 				if ( isUserInExperiment() && !isUserInControlGroup() ) {
 					$('.mw-editsection').off( '.ve-target' ).on( 'click.ve-target', init.onEditTabClick.bind( init, 'visual' ) );
 					$('.page-side-edit').off( '.ve-target' ).on( 'click.ve-target', init.onEditTabClick.bind( init, 'visual' ) );
+
+					const wrapperNode = document.querySelector('.highlight__actions');
+					const config = { childList: true, subtree: true };
```

Comment:
**question:** Do we need to have `subtree` here? Do we expect new nodes to appear deeper than on child level of .highlight__actions? Maybe we could observe another element instead?

Reply:
It was deeper but I see this works without subtree. Changed